immuni_exposure_ingestion/tasks/process_uploads.py

```diff
@@ -60,70 +66,154 @@ async def _process_uploads() -> None:
     # Acquire a lock on redis before processing anything, avoiding concurrent tasks.
     async with lock_concurrency("process_uploads"):
         _LOGGER.info("Obtained lock.")
+        _LOGGER.info("Start processing italian Teks.")
+        _batch_it()
+        _LOGGER.info("End processing italian Teks.")
```

Comment:
_minor_: I'd move the logs within the callee. Same below for the EU part. 👍

Reply:
we need the return in the `set_highest_risk_level(upload.keys)` since we use it in the method `_create_batch_eu`:
```
        keys += set_highest_risk_level(upload.keys)
        processed_uploads.append(upload.id)
```

Reply:
Though, it is the same as:
```python
        set_highest_risk_level(upload.keys)
        keys += upload.keys
        processed_uploads.append(upload.id)
```

Minor.

Reply:
done, sorry you were right